e2e-tests/gatsby-static-image/cypress/integration/intesection-observer.js

```diff
@@ -8,10 +8,14 @@ describe(`gatsby-plugin-image / intersection observer`, () => {
   })
 
   it(`lazy loads an image when scrolling`, () => {
+    cy.window().then(win => {
+      expect(win.HTMLImageElement.prototype.loading).to.equal(undefined)
```

Comment:
```suggestion
      expect("loading" in win.HTMLImageElement.prototype).toBeFalsy()
```
This matches the check in the component